openapi/components/schemas/ReportRevenueWaterfall.yaml

```
@@ -13,7 +13,7 @@ items:
       format: double
       example: 25.0
     recognized:
-      description: The revenue amount regognized up to `recognizedTo` month.
+      description: The revenue amount recognized up to `recognizedTo` month.
```

question: does recognized revenue include the `recognizedTo` month?
```suggestion
      description: Recognized revenue amount up to, and including, the `recognizedTo` month.
```

```suggestion
      description: Recognized revenue amount up to, and excluding, the `recognizedTo` month.
```